langchain4j-weaviate/src/main/java/dev/langchain4j/store/embedding/weaviate/WeaviateEmbeddingStore.java

```diff
@@ -65,7 +75,8 @@ public WeaviateEmbeddingStore(
             Integer port,
             String objectClass,
             Boolean avoidDups,
-            String consistencyLevel
+            String consistencyLevel,
+            List<Property> properties
```

Comment:
```suggestion
            Collection<String> metadataKeys
```
To keep it consistent with other integrations (e.g. redis).

Also, is it absolutely necessary? Is there a way to store/retrieve metadata without specifying explicitly each key?